src/ImageSharp.Drawing/Processing/SolidBrush{TPixel}.cs

```diff
@@ -91,9 +91,18 @@ internal override void Apply(Span<float> scanline, int x, int y)
             {
                 Span<TPixel> destinationRow = this.Target.GetPixelRowSpan(y).Slice(x, scanline.Length);
 
-                MemoryAllocator memoryAllocator = this.Target.MemoryAllocator;
+                MemoryAllocator memoryAllocator = this.Target.MemoryAllocator;
+
+#if DEBUG
+                for (int i = 0; i < scanline.Length; i++)
+                {
+                    Guard.MustBeBetweenOrEqualTo(scanline[i], 0, 1, nameof(scanline));

```

Comment:
Do we want to throw?

Reply:
Ah, that one is no longer neccessary if the pixels do the clamp, I'll remove it now